spec/std/system_spec.cr

```diff
@@ -18,4 +18,13 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns current session login" do
+      shell_login = `logname`.strip
+      shell_login = nil if shell_login.match(/no login name/)
```

Comment:
Please check the exit status, not an arbitrary error message. POSIX specifications demand to provide *a diagnostic message* but it might not always be (or contain) `no login name`. But `logname` must return non-zero exit status if no login is available - thats easier and faster anyway.